modules/claims_api/config/schemas/v2/526.json

```diff
@@ -134,917 +135,1033 @@
+            "disabilityActionType",
+            "name"
+          ],
+          "properties": {
+            "name": {
+              "description": "Name of the disability. The /disabilities endpoint on the Benefits Reference Data API may be used to retrieve possible disability names.",
+              "type": "string",
+              "example": "PTSD (post traumatic stress disorder)"
+            },
+            "classificationCode": {
+              "type": "string",
+              "description": "Classification code for the associated body system. Must match an active code returned by the /disabilities endpoint on the Benefits Reference Data API.",
+              "example": "249470"
+            },
+            "serviceRelevance": {
+              "description": "Explanation of how the disability(ies) relates to the in-service event/exposure/injury.",
+              "type": "string",
+              "example": "Heavy equipment operator in service."
+            },
+            "approximateDate": {
+              "description": "Approximate date disability began. Date must be in the past. \n Format can be either MM-DD-YYYY or MM-YYYY",
+              "type": "string",
+              "pattern": "^(0[1-9]|1[0-2])(-(0[1-9]|[1-2][0-9]|3[0-1]))?-\\d{4}$|^(0[1-9]|1[0-2])-\\d{4}$",
+              "example": "03-12-2019 or 03-2019"
+            },
+            "ratedDisabilityId": {
+              "description": "When submitting a contention with action type INCREASE, the previously rated disability id may be included.",
+              "type": "string",
+              "example": "1100583"
+            },
+            "diagnosticCode": {
+              "description": "The diagnosticCode is required if disabilityActionType is NONE and there are secondary disabilities included with the primary. The diagnosticCode must match an existing rated disability if disabilityActionType is NONE and there are secondary disabilities included with the primary.",
+              "type": "integer",
+              "example": 9999
+            },
+            "disabilityActionType": {
+              "description": "The status of the current disability.  \n If value is \"INCREASE\", then \"ratedDisabilityId\" and \"diagnosticCode\" are required.",
+              "type": "string",
+              "enum": [
+                "NONE",
+                "NEW",
+                "REOPEN",
+                "INCREASE"
+              ],
+              "example": "NEW"
+            },
+            "secondaryDisabilities": {
+              "description": "Identifies the Secondary Service Disability information of the Veteran.",
+              "type": "array",
+              "minItems": 1,
+              "uniqueItems": true,
+              "items": {
+                "type": "object",
+                "additionalProperties": false,
+                "required": [
+                  "name",
+                  "disabilityActionType",
+                  "serviceRelevance"
+                ],
```

Comment:
Doesn't the required block here need to be removed as per [API-16232](https://jira.devops.va.gov/browse/API-16232) ?